config/initializers/session_store.rb

```diff
@@ -18,4 +18,10 @@
 
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store(:cookie_store, key: '_bbb_lti_broker_session', expire_after: 60.minutes)
+attrs = {
+  key: '_bbb_app_broker_session',
```

Comment:
Also the name of the key should be _bbb_lti_broker_session as it is based on the name of the app